pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -37,8 +37,8 @@ contract MerkleOrchard {
     // channelId > balance
     mapping(bytes32 => uint256) private _suppliedBalance;
 
-    event DistributionAdded(address indexed token, uint256 amount);
-    event DistributionSent(address indexed user, address indexed token, uint256 amount);
+    event DistributionAdded(bytes32 indexed channelId, IERC20 indexed token, bytes32 merkleRoot, uint256 amount);
```

Comment:
I think I prefer `event DistributionAdded(address indexed distributor, IERC20 indexed token, bytes32 merkleRoot, uint256 amount);` as `channelId` is just an implementation detail irrelevant to the claimant and this would facilitate better filtering based on the address adding rewards